build.gradle

```diff
@@ -17,7 +17,8 @@
  */
 
 buildscript {
-  ext.kotlin_version = '1.5.30'
+  ext.kotlin_version = '1.5.21'
```

Comment:
CI won't pass without this. We probably need to delay it until Compose stable for 1.5.30 is released, or switch to 1.1.0-alpha03.